### PR TITLE
added toast

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
127 0 0 1_5500_Documentation_toast_toast html

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Small PR. Toast looks good.

File: Documentation/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <footer class="footer">
+      <div class="centered container">
+        <ul class="list-non-bullet">
+          <li class="list-inline">
+            <a
+              class="links"
+              target="_blank"
+              href="https://github.com/Priyanshi-Tiwari"
+              ><i class="fa-brands fa-github" style="font-size: 40px"></i
+            ></a>
```
Comment (raman1999) [inline Documentation/toast/toast.html:102]: avoid using this inline styles

File: Documentation/toast/toast.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          </li>
+
+          <li class="list-inline">
+            <a
+              class="links"
+              target="_blank"
+              href="https://www.linkedin.com/in/priyanshi-tiwari-90594a221/"
+              ><i class="fa-brands fa-linkedin-in" style="font-size: 40px"></i>
+            </a>
```
Comment (raman1999) [inline Documentation/toast/toast.html:111]: here also inline

**Review by raman1999 — COMMENTED** [review]
Looks good to me, just avoid using inline styles

File: Documentation/toast/toast.html (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply Documentation/toast/toast.html:102]: okay

File: Documentation/toast/toast.html (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply Documentation/toast/toast.html:111]: okay

**Pull request merged**